tramp-hdfs.el

```diff
@@ -237,11 +228,11 @@ Optional argument ARGS is a list of arguments to pass to the OPERATION."
   (let ((response
 	 (if tramp-hdfs-curl-path
 	     (progn
-	       (tramp-message vec 10 "Command: %s -k -sSL --negotiate -u : -X %s %s" tramp-hdfs-curl-path method url)
+	       (tramp-message vec 10 "Command: %s -k -sSL -X %s %s" tramp-hdfs-curl-path method url)
 	       (string-trim (with-output-to-string
 			      (with-current-buffer
 				  standard-output
-				(call-process tramp-hdfs-curl-path nil t nil "-k" "-sSL" "--negotiate" "-u" ":"  "-X" method url)))))
+				(call-process tramp-hdfs-curl-path nil t nil "-k" "-sSL" "-X" method url)))))
```

Comment:
How will this work with secure hdfs ?

Reply:
I think the proper thing to do here is to extract options to a separate parameter - will do it if you don't mind.
I didn't test with secure hdfs, but had some problem with simple local setup - first the macports' curl is not built with SPNEGO (and doesn't have --negotiate), second, I couldn't connect to non-secure hdfs even using curl with spnego.
So it seems logical to allow a user to customize curl invocation.